### ISSUE TITLE
Mautic API Returns 404 even after deleting cache

### ISSUE THREAD

**Issue by gtriak8** [mautic/mautic#6920]
Hi,

I am trying to call Mautic's API, it just returns a 404 error.
http://ip_address/api/contacts

I tried deleting the cache folder as suggested on so many posts from google results.
I also tried changing the mautic cache folder location outside the installation folder.

I am using Basic Auth.

Appreciate any help.

```
import requests
import base64

encoded = base64.b64encode(b'user:password').decode('utf-8')
print(encoded)

HEADERS = {'Authorization': 'Basic ' +encoded +' Cache-Control: no-cache' }
r = requests.post('http://ip_address/api/contacts', headers=HEADERS)
print(r.status_code)
```

Thanks,

**Comment by Woeler** [comment]
Basic auth is turned on I assume, so is the api? Also, what is the url configured in the Mautic install? You should run it on a hostname.

**Comment by gtriak8** [comment]
> Basic auth is turned on I assume, so is the api? Also, what is the url configured in the Mautic install? You should run it on a hostname.

Yes, Both the API and Basic Auth is On. As for the hostname, does it really matter? Mautic is already hosted on an AWS Server not on my local machine. I am using its public ip to access the API.

Appreciate the response.

**Comment by carloshmm** [comment]
> You should access it with the host specified in the Mautic settings.

Thank you! I was getting the same error message reported here and I solved this by looking to the host I specified on my Mautic configuration and I was using: http://mautic.myhost.com and the address actually had to be https://mautic.myhost.com, I forget to add the 's'. I don't know why this happened because my Apache server is configured to redirect http to https always.
